solution/diverse_word.rb

```diff
@@ -0,0 +1,41 @@
+class DiverseWord
+  def initialize(a, b, c)
+    @a = a
+    @b = b
+    @c = c
+  end
+
+  def solution
+    ans = ''
+    a = @a
+    b = @b
+    c = @c
+    con_a = 0
+    con_b = 0
+    con_c = 0
+    while a != 0 || b != 0 || c != 0
+      if (a >= b && a >= c && con_a < 2) || (con_b == 2 && a >= 1) || (con_c == 2 && a >= 1)
```

Comment:
Leave a line break before block to improve readability.
```suggestion
    con_b = 0
    con_c = 0

    while a != 0 || b != 0 || c != 0
      if (a >= b && a >= c && con_a < 2) || (con_b == 2 && a >= 1) || (con_c == 2 && a >= 1)
```